test/topology_experimental_raft/suite.yaml

```diff
@@ -18,6 +18,7 @@ skip_in_release:
   - test_raft_cluster_features
   - test_raft_ignore_nodes
   - test_topology_upgrade
+  - test_topology_upgrade_stuck
```

Comment:
You could use the new skip marker instead of this, much more convenient, can be applied per test case (instead of file) and survives renames.

Reply:
Do you have something particular in mind? I was looking for annotations like `@pytest.mark.something` but couldn't find anything in our codebase which would skip conditionally.

Reply:
Ok, I found `@skip_mode`.